src/software/utils/main_sfmTransform.cpp

```diff
@@ -216,6 +219,7 @@ int aliceVision_main(int argc, char **argv)
         "\t- transformation: Apply a given transformation\n"
         "\t- manual: Apply the gizmo transformation\n"
         "\t- auto_from_cameras: Use cameras\n"
+        "\t- auto_from_cameras_x_axis: Use cameras X axis\n"
```

Comment:
```suggestion
        "\t- auto_from_cameras_x_axis: Align the scene so that the X-axis of the cameras are, on average, parallel to the X-axis of the scene\n"
```
somethng like that?